packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.styles.ts

```diff
@@ -0,0 +1,203 @@
+import { memoizeFunction } from '../../Utilities';
+import { IContextualMenuStyles, IMenuItemStyles } from './ContextualMenu.Props';
+import {
+  ITheme,
+  IRawStyle,
+  concatStyleSets,
+  FontWeights,
+  getFocusStyle
+} from '../../Styling';
+
+const MS_HIGHCONTRAST_ACTIVE = '@media screen and (-ms-high-contrast: active) &';
+
+const ContextualMenuItemHeight = '32px';
+const ContextualMenuIconWidth = '14px';
```

Comment:
should these not be scaling with font size?

Reply:
That's just a port from the existing SASS rules. It's probably something worth following up on in future though